src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -320,19 +320,21 @@ const renderPricing = (salePrice, user, mediator, size) => {
     )
   } else {
     const btnSize = size === "xs" || "sm" ? "small" : "large"
+    const buttonMargin = size === "xs" ? "10px" : "0px"
```

Comment:
```suggestion
    const buttonMargin = size === "xs" ? 1 : 0
```

It's good to use the spacing units from palette instead of exact pixel sizes when it matches up.